urllib3/exceptions.py

```diff
@@ -180,6 +180,14 @@ class SNIMissingWarning(HTTPWarning):
     pass
 
 
+class DependencyWarning(HTTPWarning):
```

Comment:
Yeah, my "Bleh" was a remark of discontent because I don't see a good way of "fixing" this. It's not broken, just semantically irksome to me.